### PR TITLE
DOC: Fix quotes position in Timestamp and Timedelta

### PR DESCRIPTION
**Pull request by diogo-bastos**
The validate_docstrings.py validator was returning errors for methods with no comments. I only worked on the methods that already had comments in this PR.

- works on #24070

### CODE REVIEW

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 1]
```diff
@@ -253,8 +253,10 @@ def array_to_timedelta64(object[:] values, unit='ns', errors='raise'):
 
 
 cdef inline int64_t cast_from_unit(object ts, object unit) except? -1:
-    """ return a casting of the unit represented to nanoseconds
-        round the fractional part of a float to our precision, p """
+    """
+	return a casting of the unit represented to nanoseconds
+        round the fractional part of a float to our precision, p
+	"""
```
Comment (jreback) [inline pandas/_libs/tslibs/timedeltas.pyx]: this is not aligned here

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 2]
```diff
@@ -739,9 +739,18 @@ cdef _to_py_int_float(v):
 # heavy lifting.
 cdef class _Timedelta(timedelta):
     cdef readonly:
-        int64_t value      # nanoseconds
-        object freq        # frequency reference
-        bint is_populated  # are my components populated
+        int64_t value
```
Comment (jreback) [inline pandas/_libs/tslibs/timedeltas.pyx]: huh? this is so much harder to read, just revert this

**Review by jreback — CHANGES_REQUESTED** [review]
lots of alignment issues with quotes

**Comment by datapythonista** [comment]
@diogo-bastos some extra information that it was probably not clear in the issue description.

The goal of the issue is to fix the format of the docstrings, and leave them consistent everywhere. For a specific subset of docstrings.

So, regarding the issue, the only change needed is to make sure the The opening and closing quotes are correctly aligned.

So, as the issue says, this is correct:
```
def foo():
    """
    Summary of the docstring.

    Returns
    -------
    str
        Explaining what is being returned.
    """
```

and any other variation is not:
```
def foo():
    """    Summary of the docstring.

    Returns
    -------
    str
        Explaining what is being returned.
    """
```
or
```
def foo():
    """
       Summary of the docstring.

    Returns
    -------
    str
        Explaining what is being returned.
                """
```
etc.

And only for the docstrings reported by the script. So, the call to the script in the issue doesn't report any case once this is finished.

For this PR we don't expect any change to the code, change to other docstrings, change to the content of the docstrings or any unrelated change. If you think something else needs to be changed, open an issue, wait for the discussion on whether it's a good idea, and then open a separate PR specific to that issue. We need to do this to keep pandas development under control, be able to easily revert changes, and not have a chaos where we can't react to something breaking.

The only other thing that would be acceptable as part of this PR, would be to capitalize the summary and add a period at the end.

So, for example:
```
def foo():
    """ summary of the docstring """
```
could become
```
def foo():
    """
    Summary of the docstring.
    """
```

I'm going to close this for now, as I'm not sure if it'll be easier for you to start again in a new PR, or fix this. Feel free to reopen when you have the correct changes, or open a new PR.

Thanks a lot for working on this.

**Comment by diogo-bastos** [comment]
Hello @datapythonista , thank you for your feedback and patience. I will open a new PR with the correct changes soon.

**Pull request closed without merging**